WtWSMS/history/provinces/bp_k_mauretania.txt

```diff
@@ -252,7 +251,7 @@
 ###c_satif
 4639 = {	#SATIF
 	culture = taqbaylit
-	religion = donatism
+	religion = mithraic_pagan #[1]
```

Comment:
The source does not mention Mithraism, it says the city was Christian

Reply:
@Symmachus said that source mentioned mithraism. Given my poor French, I took his word for it. I'll revert.

Reply:
> @Symmachus said that source mentioned mithraism. Given my poor French, I took his word for it. I'll revert.

Well, then it turn to be another hoax from the Wiki, saying that there was Mithraic cult in the city of Sitifis, without the clear dating where it had been supplanted by Christianity. Next time I will check the source itself.

Reply:
No problems, it happens